src/lib/models/validators.js

```diff
@@ -78,6 +79,8 @@ const assertQuantityWithMaximum = (value, max) => assert(value, VALID_QUANTITY.m
 const assertNullableQuantity = value => assert(value, VALID_NULLABLE_QUANTITY);
 const assertIsoString = value => assert(value, VALID_ISO_DATE_STRING);
 const assertFactor = value => assert(value, VALID_FACTOR);
+const assertFactorWithPrefix = value => assert(value, VALID_FACTOR_WITH_PREFIX);
+const assertNumeric = value => assert(value, Joi.number().allow(0).allow(null));
```

Comment:
if a value can be null we add nullable i.e.  `const assertNullableNumeric = ...`